internal/daemon/vendor-specific-plugins/common/vspnetutils.go

```diff
@@ -110,3 +132,134 @@ func GetNetDevNameFromPCIeAddr(platform platform.Platform, pcieAddress string) (
 
 	return "", fmt.Errorf("network device not found for PCI address %s", pcieAddress)
 }
+
+// CreateVethPair function to create a veth pair with the given index and InterfaceInfo
+func CreateVethPair(index int) (*VEthPairDeviceInfo, error) {
+	//nfInterfaceName is the name of the interface on the Network Function attached to the container
+	//dpInterfaceName is the name of the interface on the Data Plane OvS side
+	nfInterfaceName := fmt.Sprintf("nf_if%d", index)
+	dpInterfaceName := fmt.Sprintf("dp_if%d", index)
+
+	var nfLink netlink.Link
+	var dpLink netlink.Link
+	var err error
+
+	// Create early for clean up purposes.
```

Comment:
This comment seems unnecessary and explains nothing that is not already visible in the code. `deviceInfo` is used in the very next line with `DestroyVethPair(&deviceInfo)`.

Reply:
`DestroyVethPair()` only uses `dev.NfInterfaceName` (which probably makes sense). As it is, `DestroyVethPair()` would be better to only take a `peer_ifname string` argument. Then you can call `DestroyVethPair(nfInterfaceName)`.

Reply:
I think it is a better API to pass the data structure that represents VEthPairs, such that the objects can be tracked and followed everywhere without exceptions.

Reply:
I can remove that comment

Reply:
DestroyVethPair can be used outside which passing the object in would be better.